packages/app/src/types.ts

```diff
@@ -25,35 +25,7 @@ export interface PlatformLoggerService {
 }
 
 export interface HeartbeatService {
-  // The persistence layer for heartbeats
-  storage: HeartbeatStorage;
-  /**
-   * in-memory cache for heartbeats, used by getHeartbeatsHeader() to generate the header string.
-   * Populated from indexedDB when the controller is instantiated and should be kept in sync with indexedDB
-   */
-  heartbeatsCache: HeartbeatsByUserAgent[] | null;
-
-  /**
-   * the initialization promise for populating heartbeatCache.
-   * If getHeartbeatsHeader() is called before the promise resolves (hearbeatsCache == null), it should wait for this promise
-   */
-  heartbeatsCachePromise: Promise<HeartbeatsByUserAgent[]>;
-
-  /**
```

Comment:
The comments are still useful.

Reply:
I put the comments on the class itself, do they need to be in both places?

Reply:
I think so. The consumer of the component will only look at the interface definition.

Reply:
Restored.